components/filter/README.md

```diff
@@ -220,12 +221,43 @@ This component is built to be used alongside the [d2l-filter-dimension-set](#d2l
 
 | Property | Type | Description |
 |---|---|---|
+| `count` | Number, default: `undefined` | Count value for the value in the list |
```

Comment:
For this one here, I would exclude the default, and in the description I would say "Count for the value in the list. If no count is provided, no count will be displayed." I would update the comment in the code to be this as well, since that's what gets used when the docs are generated.

Reply:
Changes made.